docs/core-library/h3indexing.md

```diff
@@ -34,10 +34,12 @@ The components of the **H3** unidirectional edge index (mode 2) are packed into
 * 7 bits to indicate the base cell 0-121, and
 * 3 bits to indicate each subsequent digit 0-6 from resolution 1 up to the resolution of the cell (45 bits total is reserved for resolutions 1-15)
 
-The canonical string representation of an **H3Index** is the hexadecimal representation of the integer.
+The canonical string representation of an **H3Index** is the hexadecimal representation of the integer, using lowercase letters.
```

Comment:
Should we also specify that leading zeros should be excluded? For example, cells can be represented with 15 hex digits, like `89754e64993ffff`. I could imagine a user assuming that the hex representation would **always** be 16 hex digits, and might pad with a leading zero: `089754e64993ffff`.

This would correspond to the same cell, but the strings would not match.